kong/cmd/utils/nginx_signals.lua

```diff
@@ -52,16 +52,15 @@ function _M.find_bin()
     local path_to_check = pl_path.join(path, nginx_bin_name)
     if is_openresty(path_to_check) then
       found = path_to_check
+      log.verbose("found OpenResty 'nginx' executable at %s", found)
       break
     end
   end
 
   if not found then
-    return nil, "could not find OpenResty 'nginx' executable"
+    return nil, ("could not find OpenResty 'nginx' executable. Kong requires version %s"):format(tostring(nginx_compatible))
```

Comment:
output
````
2016/08/08 17:34:40 [verbose] searching for OpenResty 'nginx' executable...
2016/08/08 17:34:40 [verbose] OpenResty 'nginx' executable not found at /usr/local/openresty/nginx/sbin/nginx
2016/08/08 17:34:40 [verbose] incompatible nginx found at nginx. Kong requires OpenResty, version 1.9.15.1, got 1.9.7.5
Error:
./kong/cmd/start.lua:21: could not find OpenResty 'nginx' executable. Kong requires version 1.9.15.1
stack traceback:
	[C]: in function 'assert'
	./kong/cmd/start.lua:21: in function 'cmd_exec'
	./kong/cmd/init.lua:81: in function <./kong/cmd/init.lua:81>
	[C]: in function 'xpcall'
	./kong/cmd/init.lua:81: in function <./kong/cmd/init.lua:42>
	./bin/kong:3: in main chunk
	[C]: in function 'pcall'
	init_worker_by_lua:38: in function <init_worker_by_lua:36>
````
So yes, twice the required Kong version. We could ditch the first one.

But because a list of locations is tested, we cannot throw the error from the check, as it chould continue to the next option. Only if all paths fail, an error is thrown.